src/platform/baytrail/include/platform/lib/clk.h

```diff
@@ -15,6 +15,8 @@
 #include <sof/lib/shim.h>
 #include <stdint.h>
 
+struct sof;
```

Comment:
I wonder if we should just declare this in a global "context" header ?

Reply:
@lgirdwood I don't think we should forward it globally. Every header should declare things for itself.

Reply:
I'm thinking along the lines of linux/kernel.h, but this can be looked at later.